packages/vite/src/plugins/plugin.ts

```diff
@@ -127,7 +127,13 @@ async function buildViteTargets(
   const targets: Record<string, TargetConfiguration> = {};
 
   // If file is not vitest.config and buildable, create targets for build, serve, preview and serve-static
-  if (!configFilePath.includes('vitest.config') && isBuildable) {
+  const hasRemixPlugin =
+    viteConfig.plugins && viteConfig.plugins.some((p) => p.name === 'remix');
```

Comment:
1. Super cool, I didn't noticed we have access to the entire parsed config!
2. Should we check if the plugin name includes `remix` rather than being an exact match?

Reply:
@SeanSanker FYI `@nx/remix` update to support vite https://github.com/nrwl/nx/pull/22757
1. In `@nx/vite` yes, there are some handy utils. Not available in `@nx/remix`, though there are some code that could be reuse at https://github.com/nrwl/nx/blob/master/packages/remix/plugins/component-testing/index.ts (working on that)
2. Yes, @Coly010 said so

Reply:
@SeanSanker 
for point 2: Not in this case. Remix's official vite plugin sets the plugin name to `remix` exactly.
If someone published a vite plugin such as `remixhelmet`, which may have nothing to do with remix.run itself, with the name set to this, then `p.name.includes("remix")` would return true for `remixhelmet` creating a potentially invalid result.

Reply:
Oh, sorry, I totally misunderstood the question